src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
+// // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const maxItem = currentPage * itemsPerPage;
+  const firstItem = maxItem - itemsPerPage + 1;
+  const lastItem = maxItem > items.length ? items.length : maxItem;
+
+  const visibleItems = items.slice(firstItem - 1, lastItem);
+
+  const pageChange = (page: number) => {
```

Comment:
```suggestion
  const handlePageChange = (page: number) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364